lists/eg.csv

```diff
@@ -175,3 +175,88 @@ http://www.alarab.qa,NEWS,News Media,2017-05-24,OONI,blocked as of date added
 http://www.raya.com/portal,NEWS,News Media,2017-05-24,OONI,blocked as of date added
 https://hasamegypt.com,NEWS,News Media,2017-05-24,OONI,blocked as of date added
 http://www.aswatmasriya.com,NEWS,News Media,2017-05-24,OONI,blocked as of date added
+http://6april.org,POLR,Political Criticism,2017-06-23,AFTE,
+http://akhbaralaalam.net,NEWS,News Media,2017-06-23,AFTE,
+http://akhbaralsabah.com,NEWS,News Media,2017-06-23,AFTE,
+http://akhbarturkiya.com,NEWS,News Media,2017-06-23,AFTE,
+http://alalam.ir,NEWS,News Media,2017-06-23,AFTE,
+http://alaraby.tv,NEWS,News Media,2017-06-23,AFTE,
+http://albedaiah.com,NEWS,News Media,2017-06-23,AFTE,
+http://alkessa.com,NEWS,News Media,2017-06-23,AFTE,
+http://almanassa.com,NEWS,News Media,2017-06-23,AFTE,
+http://almesryoon.com,NEWS,News Media,2017-06-23,AFTE,
+http://amlalommah.net,NEWS,News Media,2017-06-23,AFTE,
+http://ansarportsaed.com,NEWS,News Media,2017-06-23,AFTE,
+http://arab-turkey.com,NEWS,News Media,2017-06-23,AFTE,
+http://avaaz.org,MISC,Miscelaneous content,2017-06-23,AFTE,
+http://balkans.aljazeera.net,NEWS,News Media,2017-06-23,AFTE,
+http://blogs.aljazeera.net,NEWS,News Media,2017-06-23,AFTE,
+http://cyberghostvpn.com,ANON,Anonymization and circumvention tools,2017-06-23,AFTE,
+http://doc.aljazeera.net,NEWS,News Media,2017-06-23,AFTE,
+http://elbadil.com,NEWS,News Media,2017-06-23,AFTE,
+http://elbehira.net,NEWS,News Media,2017-06-23,AFTE,
+http://elsharq.web.tv,NEWS,News Media,2017-06-23,AFTE,
+http://hamas.ps,POLR,Political Criticism,2017-06-23,AFTE,
+http://harmees.com,NEWS,News Media,2017-06-23,AFTE,
+http://hasamegypt.com,MILX,Terrorism and Militants,2017-06-23,AFTE,
+http://ida2at.com,NEWS,News Media,2017-06-23,AFTE,
+http://institute.aljazeera.net,NEWS,News Media,2017-06-23,AFTE,
+http://kh-press.com,NEWS,News Media,2017-06-23,AFTE,
+http://learning.aljazeera.net,NEWS,News Media,2017-06-23,AFTE,
+http://madaad.net,NEWS,News Media,2017-06-23,AFTE,
+http://marsadpress.net,NEWS,News Media,2017-06-23,AFTE,
+http://masralekhbaria.com,NEWS,News Media,2017-06-23,AFTE,
+http://masreiat.com,NEWS,News Media,2017-06-23,AFTE,
+http://medium.com,NEWS,News Media,2017-06-23,AFTE,
```

Comment:
this should probably land in the global list. also I believe medium.com has https, so the url should be specified with https://